### PR TITLE
Added new Resource Group as per Task#1234

### PR DESCRIPTION
**Pull request by devopsinsiders**
My manager asked me to add the new rg so I have added that and raised a Pull request.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -6,4 +6,9 @@ resource "azurerm_resource_group" "rg-dhondhu" {
 resource "azurerm_resource_group" "rg-jackie" {
   name     = "rg-jackie"
   location = "West Europe"
+}
+
+resource "azurerm_resource_group" "rg-rahul" {
+  name     = "rg-rahul"
+  location = "CentralIndia"
```
Comment (rahulsharma200990) [inline main.tf]: yeh galat hai. location sahi karo location us ki honi chayee

Comment (rahulsharma200990) [reply main.tf]: sahi s check karo aap

Comment (devopsinsiders) [reply main.tf]: Done sir Location updated to west us

Comment (rahulsharma200990) [reply main.tf]: Ab theek hai

**Review by rahulsharma200990 — APPROVED** [review]
Ab sab shai h . next time dhyan rakhana

**Comment by mukesh9709** [comment]
Approved

**Review by inframellow — APPROVED** [review]
Approved. Well Done.

**Review by imprabhatranjan — APPROVED** [review]

**Comment by Srijan-DevOpsEr** [comment]
Gajab , Kya code banaya hai.. Well Done

**Review by Chiranjibi-P — APPROVED** [review]
👍👍

**Review by Chikanemukesh — APPROVED** [review]

**Review by mukesh9709 — APPROVED** [review]
Code sahi hai

**Pull request merged**